src/com/sun/javatest/exec/ResultModifierInterface.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * $Id$
+ *
+ * Copyright (c) 2002, 2012, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.  Oracle designates this
+ * particular file as subject to the "Classpath" exception as provided
+ * by Oracle in the LICENSE file that accompanied this code.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+package com.sun.javatest.exec;
+
+import com.sun.javatest.Status;
+import com.sun.javatest.TestDescription;
+
+/**
+ * Enables users to create their own result modifications for purposes of different testing strategies.
+ * This allows user to omit certain kinds of failures or pinpoint tests that passed even though the jvm
+ * crashed in the process without being intended to.
+ */
+public interface ResultModifierInterface {
```

Comment:
```suggestion
public interface ResultModifier {
```

Should it be just `ResultModifier` rather than `ResultModifierInterface`?

Reply:
I am under impression that naming interface with suffix "Interface" is kinda a good practice..

Reply:
but if you insist on renaming it I have no problem with it :-)

Reply:
Neither jtreg nor openjdk use `...Interface` suffixes. Java naming convention does not suggest using it as well: https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html

The naming of `ResultModifier` does not correspond to the actual intention of the interface: the implementation does not **modify** the result.

Reply:
Okay, will be renamed as `StatusTransformator` ?

Reply:
renamed as StatusTransformer.. Transformator has been evoking the electronic component name in my brain and did not seem correct.. also changer seems suitable, but that is up to you

Reply:
`StatusTransformator` looks great to me

Reply:
`StatusTransformer` looks great to me